src/DrawerLayout.js

```diff
@@ -212,7 +212,7 @@ export default class DrawerLayout extends Component {
   }
 
   _shouldSetPanResponder = (e, { moveX, dx, dy }) => {
-    if (!dx || !dy) {
+    if (!dx || !dy || Math.abs(dx) < 3) {
```

Comment:
First of all thank you for your time and help @pyoung213 and @satya164, it is highly appreciated! I like the solution in general and would love to give it a try (it will be better than the current state and in case of new issues we could also revert this change and give the other solution a try)

Regarding the pixel density thing, can we use [PixelRation](https://facebook.github.io/react-native/docs/pixelratio.html) here?
Also I would like to remove the `3`somehow, either put it in a constant with a small comment above why it is chosen & what it is for or we could expose it as a prop with default props set (and a comment there)